databricks/koalas/series.py

```diff
@@ -3529,7 +3529,42 @@ def __len__(self):
         return len(self.to_dataframe())
 
     def __getitem__(self, key):
-        return self._with_new_scol(self._scol.__getitem__(key))
+        if not isinstance(key, (str, tuple)):
+            raise ValueError("'key' should be string or tuple that contains strings")
+        if isinstance(key, str):
+            key = (key,)
+        if not all(isinstance(index, str) for index in key):
+            raise ValueError("'key' should have index names as only strings "
+                             "or a tuple that contain index names as only strings")
+        if len(self._internal._index_map) < len(key):
+            raise KeyError("Key length ({}) exceeds index depth ({})"
+                           .format(len(key), len(self._internal.index_map)))
+
+        cols = (self._internal.index_scols[len(key):] +
+                [self._internal.scol_for(self._internal.column_index[0])])
+        rows = [self._internal.scols[level] == index
+                for level, index in enumerate(key)]
+        sdf = self._internal.sdf \
+            .select(cols) \
+            .where(reduce(lambda x, y: x & y, rows))
+
+        if len(self._internal._index_map) == len(key):
+            # if sdf has one column and one data, return data only without frame
+            pdf = sdf.limit(2).toPandas()
+            length = len(pdf)
+            if length == 1:
+                return pdf[self.name].iloc[0]
+
+            key_string = ('(' + ', '.join(key) + ')') if len(key) > 1 else key[0]
```

Comment:
@itholic, seems you can just use direct string implementation of `key`.

Reply:
@HyukjinKwon Thanks for reviewing! you mean like this?

```python
>>> key = ('a', 'b')
>>> str(key)
"('a', 'b')"
```

if right, the result maybe like

```python
('cow', 'weight')    200.0
('cow', 'weight')      1.2
('cow', 'weight')    250.0
Name: 0, dtype: float64
```

not 

```python
(cow, weight)    200.0
(cow, weight)      1.2
(cow, weight)    250.0
Name: 0, dtype: float64
```

is it okay?

Reply:
ah, okay. let's just leave it as is. It matches with pandas' right?

Reply:
@HyukjinKwon right :)

Reply:
@ueshin sure, then first i'll make util function for make string like this in the separate PR, and after then, fix the existing strings including this one.

Reply:
i just made util function `tuple-like strings` and applied.